bigchaindb/log/configs.py

```diff
@@ -62,6 +62,7 @@
     'loggers': {},
     'root': {
         'level': logging.DEBUG,
-        'handlers': ['console', 'file', 'errors']
+        'handlers': ['console', 'file', 'errors'],
+        'port': 9020
```

Comment:
Should we use DEFAULT_SOCKET_LOGGING_PORT instead?

Reply:
Yes, that would be better.